lib/providers/services/http_service.dart

```diff
@@ -29,7 +29,7 @@ class HttpService {
   static HttpService of(BuildContext context, {bool listen = false}) => Provider.of(context, listen: listen);
 
   Future<List<dynamic>> getTableRows({String code, String scope, String table, String value}) async {
-    final requestURL = '$baseURL/v1/chain/get_table_rows';
+    final requestURL = Uri.parse('$baseURL/v1/chain/get_table_rows');
```

Comment:
This Uri.parse() Is it to add the http part ??

Reply:
the call below takes a Uri and not a string